applications/minotari_console_wallet/src/grpc/wallet_grpc_server.rs

```diff
@@ -1089,10 +1089,10 @@ async fn handle_pending_outbound(
     sender: &mut Sender<Result<TransactionEventResponse, Status>>,
 ) {
     match transaction_service.get_pending_outbound_transactions().await {
-        Ok(mut txs) => {
-            if let Some(tx) = txs.remove(&tx_id) {
```

Comment:
nit: I think it would have been better to rather use the DB. 
Either call get_any_tx with the id, or create a new call to search the db
But I think this is fine for now.